docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -185,11 +184,15 @@ use case. This means you can, for example, copy an existing CA into `/etc/kubern
 and `/etc/kubernetes/pki/ca.key`, and kubeadm will use this CA for signing the rest 
 of the certs.
 
+#### External CA mode {#external-ca-mode}
+
 It is also possible to provide just the `ca.crt` file and not the 
 `ca.key` file (this is only available for the root CA file, not other cert pairs). 
 If all other certificates and kubeconfig files are in place, kubeadm recognizes 
-this condition and activates the "ExternalCA" mode, which also
-implies the CSR signer controller in the controller manager won't be started.
+this condition and activates the "External CA" mode, which means that kubeadm will
```

Comment:
Replace through the end of the paragraph with:
```
this condition and activates the "External CA" mode. kubeadm will proceed without the CA key on disk.

**Note:** Don't run the CSR signer controller. Instead, run the controller-manager standalone with `--controllers=csrsigner` and point to the CA certificate and key.
{: .note}
```
So:
> this condition and activates the "External CA" mode. kubeadm will proceed without the CA key on disk.
>
> **Note:** Don't run the CSR signer controller. Instead, run the controller-manager standalone with `--controllers=csrsigner` and point to the CA certificate and key.
> {: .note}

Reply:
fixed